source/jormungandr/jormungandr/instance.py

```diff
@@ -213,7 +214,15 @@ def get_external_service_providers_from_db(self):
         """
         :return: a callable query of external services associated to the current instance in db
         """
-        return self._get_models().external_services
+        result = self._get_models().external_services
+        return [res for res in result if res.navitia_service != 'realtime_proxies']
```

Comment:
It seems fine now but when we add a new service, we will have to modify here and below. What I propose:
return [res for res in result if res.navitia_service in ['free_floatings', 'vehicle_occupancies']]

Reply:
OK